src/main/java/hudson/remoting/Channel.java

```diff
@@ -45,6 +45,7 @@
 import java.io.UnsupportedEncodingException;
 import java.lang.ref.WeakReference;
 import java.net.URL;
+import java.time.Duration;
```

Comment:
Could use `long` + `TimeUnit` if you wanted to keep Java 7 compatibility. (But I see no reason not to move to 8 immediately.)